tests/py_test.py

```diff
@@ -59,6 +59,57 @@ def assertRaises(test_case, fn, exception_class):
     test_case.assertTrue(exc is not None)
     test_case.assertTrue(isinstance(exc, exception_class))
 
+class AlignmentError(AssertionError):
+    def __init__(self, addr, alignment):
+        self.addr = addr
+        self.alignment = alignment
+        self.offBy = self.addr % self.alignment
+
+    def __str__(self):
+        return "Address %d is not alligned to %d. Off by %d" \
+               % (self.addr, self.alignment, self.offBy)
+
+class TestProxyTable(object):
```

Comment:
I do not understand why this is a new class, again only used in 1 place, with most of the code being boilerplate. Please inline where the test is actually done, it should collapse the code to a fraction.

Reply:
The `TestProxyTable` wraps the default `Monster._tab`. It then verifies alignment on all attribute access. The only way I can see to inline it be to move the class into `CheckReadBuffer`, or directly edit the attributes of `Monster._tab` and add the necessary methods, but the second option doesn't seem very pythonic. Any Ideas?

Reply:
https://github.com/google/flatbuffers/pull/5770#discussion_r588823184

Reply:
I don't think we should be wrapping the API at all. We want to test the API as-is, without any possible side-effects from patching the API, or making it harder to debug.

If there is anything to be tested about alignment, it can be its own, isolated test.